keyboards/geistmaschine/macropod/readme.md

```diff
@@ -0,0 +1,25 @@
+# Geistmaschine Macropod
+
+![macropod](https://i.imgur.com/lAsCslA.png)
```

Comment:
```suggestion
![macropod](https://i.imgur.com/lAsCslAh.png)
```

Reply:
Same issue as with the other PR, the picture is already scaled at 640px width, but the imgur compression removes transparency and adds some artifacts around the edges.